src/com/example/mutations.cljs

```diff
@@ -35,7 +35,10 @@
         response {response-id {:response/id response-id
                                :response/total total-score
                                :answer/id (map (fn [a] [:answer/id (key a)]) answers)}}]
-    (merge state {:answer/id answers} {:response/id response})))
+    (merge state
+           {:answer/id answers}
+           {:response/id response}
+           {:root/response [:response/id response-id]})))
```

Comment:
added the current response as the `:root/response` so that we can query for this toplevel ident and get todays response.